src/client/gui/lib/settings/virtualization_settings.dart

```diff
@@ -42,22 +45,10 @@ class VirtualizationSettings extends ConsumerWidget {
           value: networks.contains(bridgedNetwork) ? bridgedNetwork : null,
```

Comment:
I'd go with something like 'Bridged network', since when configuring the instance we have the option of 'Connect to bridged network'. That way it should be clear that they refer to the same thing.